### DISCUSSION TITLE
Ability to open a ticket with no body content

### DISCUSSION THREAD

**Discussion by mzealey** [nyaruka/rapidpro#4248] [Ideas]
It's great to see the contact -> Open Ticket functionality in the new dev branch - can't wait for it to come into a production release.

As far as I can see from testing, the 'body' content is only used internally, however it is still required. It would be nice to make it not required so an agent can open a chat with a particular user without having to type something superfluous in there. Either some pre-populated ticket such as 'No content' or allowing it to be blank.

**Comment by ericnewcomer** [comment]
The body is shown in the ticket interface. I don't see a strong argument against making it optional though.
